Swap servo images

avanheuvelen opened the pull request:
## What This PR Changes
The servo page has the images in the wrong order. The 'sweep' circuit is shown under 'knob' circuit and vice versa.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/arduino/docs-content?pullRequest=1685) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/arduino/docs-content?pullRequest=1685) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/arduino/docs-content?pullRequest=1685) it.</sub>

jhansson-ard commented:
Hello @avanheuvelen ! 

Thank you for your contribution! 

Please revert the names of the images to their originals, but keep your change of exchanging the places of the photos. I can see you are completely right. 

When you have reverted the names, and updated your PR as well as signed the CLA - we'll be able to review your PR properly.

Thank you!

avanheuvelen commented:
> Hello @avanheuvelen !
> 
> Thank you for your contribution!
> 
> Please revert the names of the images to their originals, but keep your change of exchanging the places of the photos. I can see you are completely right.
> 
> When you have reverted the names, and updated your PR as well as signed the CLA - we'll be able to review your PR properly.
> 
> Thank you!

I've reverted the filename change for you

jhansson-ard commented:
Thank you 🙏🏼 

As soon as you sign the CLA we can properly review your PR ⭐
